examples/LexTypes.h

```diff
@@ -3,6 +3,7 @@
 
 #define BUFFER_SIZE 256
```

Comment:
@muneebkq Please refer to the latest types.h file. It seems you missed including a header file `stdint.h`. This file is needed for older version of gcc as we are using int64_t.

Reply:
refactored